tools/load-testing/load-gas-loop.ts

```diff
@@ -127,12 +133,24 @@ const callContract = async (loopCount: number) => {
   throw new Error("Failed to verify contract call (timeout)");
 };
 
+const keypress = async () => {
+  process.stdin.setRawMode(true);
+  return new Promise((resolve) =>
+    process.stdin.once("data", (key) => {
+      process.stdin.setRawMode(false);
+      resolve(null);
+    })
+  );
+};
+
 const main = async () => {
   await deployContract();
-  let loopCount = 2;
+  let loopCount = 32768;
   while (loopCount > 0) {
+    console.log("Press key to call contract");
+    const key = await keypress();
+    console.log(`Key: ${key}`);
```

Comment:
What's the idea behind the keypress checking?

Reply:
I wanted to know if I could know which key it was in order to have different behavior :) will remove it